src/api/helpers/middleware.js

```diff
@@ -1,18 +1,58 @@
+/*
+ * Copyright (C) 2019  Akhilesh Kumar
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program; if not, write to the Free Software Foundation, Inc.,
+ * 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+ */
+
 
 import * as commonUtils from '../../common/helpers/utils';
 import _ from 'lodash';
+import log from 'log';
 
 
 /* eslint-disable*/
 
+export function validateAuthorBrowseRequest (req, res, next) {
```

Comment:
Please help to validate browse request here and This function return URL not valid if the request has invalid query params.